### ISSUE TITLE
Depth estimation from underwater monocular video sequences

### ISSUE THREAD

**Issue by buster-zbb** [nianticlabs/manydepth#50]
Hi@mdfirman @daniyar-niantic ,
Thanks for your work!
I tested your model in the underwater data set, but the effect is not very good. after debugging, the loss function drops normally, and the pose network can work normally, but the final result is very strange.the depth data is almost between 0.01-0.15m. I want to ask whether is the model doesn't work for this type of dataset,?here are some images from my dataset, do you know what's the problem?Thanks!
![3415](https://user-images.githubusercontent.com/84183560/163981665-8e6cf716-5373-47c0-87e2-d66d69700d0a.jpeg)
![3415_disp_multi](https://user-images.githubusercontent.com/84183560/163981747-c8d336f8-1db1-4c75-a5cb-7aff0224ba5a.jpg)
![3048](https://user-images.githubusercontent.com/84183560/163981776-f8ac6d12-c56d-4b81-9e39-f3bd9573f6b1.jpeg)
![3374](https://user-images.githubusercontent.com/84183560/163981993-16a390b6-9ff8-46c8-bce8-a61c7c7a3570.jpeg)
![4856](https://user-images.githubusercontent.com/84183560/163982103-acf273fb-871f-41e3-9377-1d002324422e.jpeg)

**Comment by daniyar-niantic** [comment]
Hi.

It is hard to say exactly what is the problem, but could you try training at lower resolution to see if that works?

**Comment by buster-zbb** [comment]
@daniyar-niantic @mdfirman ，
I tried to use the pre-trained model in kitti, the size used for training is 192*640, but the effect has not improved. I wonder if the model will work on this underwater dataset.

**Comment by daniyar-niantic** [comment]
The depthmap in your question is much higher resolution than that. How are you generating these depthmap? At test time you should use the same resolution as training resolution.

**Comment by buster-zbb** [comment]
> The depthmap in your question is much higher resolution than that. How are you generating these depthmap? At test time you should use the same resolution as training resolution.
The depth map is obtained from running test_simple.py from the code you guys provided, I think the biggest problem is that the depth value of the training output stays between 0.01-0.15m, which is very strange.
